src/app/components/team/TeamReport/TeamReportComponent.js

```diff
@@ -38,6 +38,8 @@ const TeamReportComponent = ({ team, setFlashMessage }) => {
     setReports(newReports);
   };
 
+  console.log(currentLanguage); // eslint-disable-line no-console
```

Comment:
Stray console.log

Reply:
oh yeah, thanks @amoedoamorim , forgot to take this out